### PR TITLE
[PLUGIN-1154][0.19] Fixed issue with NPE during Update/Upsert operations

### PR DESCRIPTION
**Pull request by fernst**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java [hunk 1]
```diff
@@ -589,9 +589,12 @@ private void handleInsertOperation(TableReference tableRef, String writeDisposit
       triggerBigqueryJob(projectId, jobId, dataset, config);
     }
 
-    private void handleUpdateUpsertOperation(TableReference tableRef, @Nullable String cmekKey, JobId jobId,
+    private void handleUpdateUpsertOperation(TableReference tableRef,
```
Comment (avirmani) [inline src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java:592]: Need to add the param in the calling place as well.

Comment (fernst) [reply src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java:592]: oof

Comment (fernst) [reply src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java:592]: Added now.

**Review by avirmani — APPROVED** [review]
lgtm

**Pull request merged**